assets/components/manageContributionCta/manageContributionCta.jsx

```diff
@@ -0,0 +1,29 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
+import CtaLink from 'components/ctaLink/ctaLink';
+import { getBaseDomain } from 'helpers/url';
+
+// ----- Functions ----- //
+
+// Build MMA url
+function buildUrl(): string {
```

Comment:
Maybe, renamed this function to be `buildMMAUrl` and delete the comment above it.